test/gtest/uct/ib/test_sockaddr.cc

```diff
@@ -370,12 +380,12 @@ class test_uct_cm_sockaddr : public uct_test {
 
 protected:
 
-    void cm_start_listen() {
+    void cm_start_listen(uct_listener_conn_request_callback_t server_conn_req_cb) {
```

Comment:
here and everywhere in scope of the class, since class name already has "cm", no need to have this prefix in members and methods